include/deal.II/base/lazy.h

```diff
@@ -396,5 +403,14 @@ Lazy<T>::value_or_initialize(const Callable &creator)
 }
 
 
+template <typename T>
+std::size_t
+Lazy<T>::memory_consumption() const
+{
+  return MemoryConsumption::memory_consumption(object) +
+         sizeof(object_is_initialized) + sizeof(initialization_mutex);
```

Comment:
Would
```suggestion
  return MemoryConsumption::memory_consumption(object) +
         sizeof(*this) - sizeof(object);
```
be more precise? Not that it is going to be a large difference, but due to alignment reasons, I expect that `std::atomic<bool>` is of size 1, but will likely require 8 or 16 bytes, depending on the alignment of the mutex.

Reply:
Agreed.